tools/rdm/TestMixins.py

```diff
@@ -125,16 +161,36 @@ def VerifyResult(self, response, fields):
           (self.EXPECTED_FIELD.capitalize(), self.PID,
            fields[self.EXPECTED_FIELD].encode('string-escape')))
 
+    if self.MIN_LENGTH and len(fields[self.EXPECTED_FIELD]) < self.MIN_LENGTH:
+      self.SetFailed(
+          '%s field in %s was shorter than expected, was %d, expected %d' %
+          (self.EXPECTED_FIELD.capitalize(), self.PID,
+           len(fields[self.EXPECTED_FIELD]), self.MIN_LENGTH))
+
+    if self.MAX_LENGTH and len(fields[self.EXPECTED_FIELD]) > self.MAX_LENGTH:
+      self.SetFailed(
+          '%s field in %s was longer than expected, was %d, expected %d' %
+          (self.EXPECTED_FIELD.capitalize(), self.PID,
+           len(fields[self.EXPECTED_FIELD]), self.MAX_LENGTH))
+
 class GetWithDataMixin(object):
-  """GET a PID with junk param data."""
+  """GET a PID with junk param data.
+
+    If ALLOWED_NACK is defined, this adds an additional custom NackGetResult to
```

Comment:
I'd change to ALLOWED_NACKS which is a list so we can allow more than one later if / when we need to.